src/apurvaJun20/Test3.java

```diff
@@ -0,0 +1,22 @@
+package apurvaJun20;
+
+public class Test3 {
+
+	void printPairs(int arr[], int n, int sum) {
+		int count = 0;
+		for (int index = 0; index < n; index++)
```

Comment:
logic wise good,you got the point in logic but why client will tell you length at calling it is our service method responsibility to check that right?

this is also good.
add this in your improvement part.